plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/MavenChecksumReader.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-2015 Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.repository.maven.internal;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Locale;
+
+import javax.annotation.Nullable;
+
+import org.sonatype.nexus.repository.view.Payload;
+import org.sonatype.sisu.goodies.common.ComponentSupport;
+
+import com.google.common.base.Charsets;
+import com.google.common.base.Strings;
+import com.google.common.io.CharStreams;
+import org.codehaus.plexus.util.StringUtils;
```

Comment:
TODO have a habbit of never getting TODONE, so where you can avoid them by simply not needing a TODO in this case using another non-plexus String helper like Strings from guava or String2 from nexus-common, use it.

—jason




On March 11, 2015 at 1:30:09 AM, Tamas Cservenak (notifications@github.com) wrote:

In plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/MavenChecksumReader.java:

> + */
> +package org.sonatype.nexus.repository.maven.internal;
> +
> +import java.io.IOException;
> +import java.io.InputStreamReader;
> +import java.util.Locale;
> +
> +import javax.annotation.Nullable;
> +
> +import org.sonatype.nexus.repository.view.Payload;
> +import org.sonatype.sisu.goodies.common.ComponentSupport;
> +
> +import com.google.common.base.Charsets;
> +import com.google.common.base.Strings;
> +import com.google.common.io.CharStreams;
> +import org.codehaus.plexus.util.StringUtils;
There is a TODO to remove it.

—
Reply to this email directly or view it on GitHub.